src/main/scala/viper/gobra/ast/internal/PrettyPrinter.scala

```diff
@@ -472,7 +473,14 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
       braces(space <> showExprList(exprs) <> (if (exprs.nonEmpty) space else emptyDoc))
 
   def showLit(l: Lit): Doc = l match {
-    case IntLit(v, _) => v.toString
+    case IntLit(lit, _, base) =>
+      val prefix = base match {
+        case Binary => "0b"
+        case Octal => "0o"
```

Comment:
this is inconsistent with the frontend pretty printer

Reply:
should be fixed in the last commit